pom.xml

```diff
@@ -44,7 +44,12 @@
             <artifactId>metrics-graphite</artifactId>
             <version>${metrics.version}</version>
         </dependency>
-
+        <dependency>
```

Comment:
The org.mockito:mockito-all:1.x is outdated. Use org.mockito:mockito-core:2.x instead:
```
<dependency>
    <groupId>org.mockito</groupId>
    <artifactId>mockito-core</artifactId>
    <version>2.15.0</version>
    <scope>test</scope>
</dependency>
```

Reply:
fixed